atd-cr3-api/server.py

```diff
@@ -8,13 +8,11 @@
 import boto3
 import os
 import requests
-import hashlib
```

Comment:
This was only used in verifying the secret string called `HASURA_TRIGGER_API_KEY ` that was used to know that a message was coming from our Hasura engine to our API.